### PR TITLE
TST: Change assert_approx_equal to assert_allclose

### PR DESCRIPTION
**Pull request by jbrockmendel**
remove unused imports

Replaces #5175.

### CODE REVIEW

**Comment by jbrockmendel** [comment]
@josef-pkt ping, I think issues from previous version addressed.

**Comment by jbrockmendel** [comment]
@josef-pkt this is non-tiny but pretty easy.  Getting it out of the way, in particular the unused imports, will prevent merge conflicts down the road.

File: statsmodels/regression/tests/test_glsar_gretl.py [hunk 1]
```diff
@@ -368,7 +370,7 @@ def test_all(self):
         #res2 = res.get_robustcov_results(cov_type='HC1')
         # TODO: fvalue differs from Gretl, trying any of the HCx
         #assert_almost_equal(res2.fvalue, result_gretl_g1['fvalue'][1], decimal=0) #FAIL
-        #assert_approx_equal(res.f_pvalue, result_gretl_g1['f_pvalue'][1], significant=1) #FAIL
+        #assert_array_less(res.f_pvalue, result_gretl_g1['f_pvalue'][1], rtol=1e-1) #FAIL
```
Comment (josef-pkt) [inline statsmodels/regression/tests/test_glsar_gretl.py]: don't change comments like that, the new version wouldn't be identical and so the comment might not be correct anymore

besides using assert_array_less is the wrong assert anyway.

Comment (jbrockmendel) [reply statsmodels/regression/tests/test_glsar_gretl.py]: Huh, no idea what past-me had in mind here.  Will fix.

Comment (jbrockmendel) [reply statsmodels/regression/tests/test_glsar_gretl.py]: Just pushed with fix.

**Comment by jbrockmendel** [comment]
@josef-pkt comment addressed and green.

**Comment by josef-pkt** [comment]
overall too much cleaning all over the place with a few REFactorings

merging anyway
@jbrockmendel  thank you

this is pretty much the end of cleaning for 0.10 (one or two more after this)
i.e. there are too many open issues in https://github.com/statsmodels/statsmodels/issues?q=is%3Aopen+is%3Aissue+milestone%3A0.10
to spend another month on style and code cleanup
(i.e. largely style freeze for next release)

**Pull request merged**